<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DishSelectionWindow</class>
 <widget class="QDialog" name="DishSelectionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Make Order</string>
  </property>
  <widget class="QGroupBox" name="MainDishes">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>171</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="title">
    <string>Main Dishes</string>
   </property>
   <widget class="QRadioButton" name="steak">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Steak'n Fries - 15$</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="choucroute">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Choucroute- 65$</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="lasagna">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Lasagna - 24$</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Main_None">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>None</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Starters">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>171</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="title">
    <string>Starters</string>
   </property>
   <widget class="QRadioButton" name="greek">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Greek Salad - 56$</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="soup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Soup - 2$</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="sausage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Sausage Salad - 80$</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Starter_None">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>None</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Deserts">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>171</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="title">
    <string>Deserts</string>
   </property>
   <widget class="QRadioButton" name="ice">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Ice Cream - 5$</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="tiramisu">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Tiramisu - 38$</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="bavarois">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Bavarois - 96$</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Desert_None">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>None</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Beverages">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>171</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="title">
    <string>Beverages</string>
   </property>
   <widget class="QRadioButton" name="beer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Beer - 12$</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="wine">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>French Wine - 195$</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="coca">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Coca - 5$</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Beverage_None">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>None</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="Order">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Order</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Abort">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Abort</string>
   </property>
  </widget>
  <widget class="QLabel" name="ErrorMessage">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;Comic Sans MS&quot;;
color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="text">
    <string>Please make your selection:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
